rules/code-quality/tests/Rector/FuncCall/ArrayKeysAndInArrayToArrayKeyExistsRector/RunnableTestCase.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\CodeQuality\Tests\Rector\FuncCall\ArrayKeysAndInArrayToArrayKeyExistsRector;
+
+use Nette\Utils\Strings;
+use Rector\Core\Testing\PHPUnit\AbstractRectorTestCase;
+use Rector\Core\Testing\ValueObject\SplitLine;
+use ReflectionClass;
+use Symplify\SmartFileSystem\SmartFileInfo;
+
+class RunnableTestCase extends AbstractRectorTestCase
+{
+    protected function assertOriginalAndFixedFileYieldSameResult(string $file, ?callable $assertion = null): void
+    {
+        $defaultAssertion = function (RunnableInterface $original, RunnableInterface $fixed): void {
```

Comment:
Callable type is weak type. Either use contract of no functions at all. Here should be just result assertSame()

Reply:
kk, Works for me